Bottom: cd9a58a8a75a4954c49a06c491c3c74d15576252
Top:    39c745cd25381e5adb103d2a12a4570babbaf7ac
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-17 12:06:33 -0700

Refresh of inspect

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
index 7f460234ffb..24afe8c9059 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
@@ -244,19 +244,9 @@ public:
     return result;
   }
 
-  PythonObject &operator=(const PythonObject &other) {
-    Reset(PyRefType::Borrowed, other.get());
-    return *this;
-  }
-
-  void Reset(PythonObject &&other) {
+  PythonObject &operator=(PythonObject other) {
     Reset();
-    m_py_obj = other.m_py_obj;
-    other.m_py_obj = nullptr;
-  }
-
-  PythonObject &operator=(PythonObject &&other) {
-    Reset(std::move(other));
+    m_py_obj = std::exchange(other.m_py_obj, nullptr);
     return *this;
   }
